.github/workflows/main.yml

```diff
@@ -37,5 +37,7 @@ jobs:
           # We set a flag so we can skip tests that access Netlify API
           NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
           IS_FORK: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == true }}
+      - name: Update npm config
+        run: npm config set script-shell bash
```

Comment:
Is this needed?
The shell script uses `#!/bin/bash` as a shebang, so it should be run with Bash.

Reply:
Done at https://github.com/netlify/eslint-config-node/pull/18

Reply:
Will revert this pending https://github.com/netlify/eslint-config-node/pull/18 and update `@netlify/eslint-config-node` once the PR is merged and published

Reply:
Renovate force pushed and removed the changes :angry: 
I have added the changes back and fixed it with `@netlify/eslint-config-node@0.6.1`.

Reply:
Renovate force pushed and removed the changes again :anger: 
Closing this PR and opening a new one at https://github.com/netlify/cli/pull/1542